ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -169,115 +147,57 @@ const secondaryPanel = panel => ({
-  label: 'WGS SV Genotype Quality',
-  labelHelp: 'The genotype quality (GQ) represents the quality of a Structural Variant call. Recommended SV-QG cutoffs for filtering: > 10.',
-  min: 0,
-  max: 100,
-  step: 10,
 }
 
 const PANELS = [
   INHERITANCE_PANEL,
-  {
-    [DATASET_TYPE_SV_CALLS]: null,
-    hasHgmdPermission: { [true]: HGMD_PATHOGENICITY_PANEL, [false]: PATHOGENICITY_PANEL },
-  },
-  ANNOTATION_PANEL_MAP,
-  ANNOTATION_SECONDARY_PANEL_MAP,
-  IN_SILICO_PANEL_MAP,
-  {
-    ...FREQUENCY_PANEL,
-    [DATASET_TYPE_VARIANT_CALLS]: {
-      ...FREQUENCY_PANEL,
-      fields: SNP_FREQUENCIES,
+  HGMD_PATHOGENICITY_PANEL,
+  ANNOTATION_PANEL,
+  ANNOTATION_SECONDARY_PANEL,
+  IN_SILICO_PANEL,
+  FREQUENCY_PANEL,
+  LOCATION_PANEL_WITH_GENE_LIST,
+  QUALITY_PANEL,
+]
+
+const DATASET_TYPE_PANEL_PROPS = {
+  [DATASET_TYPE_VARIANT_CALLS]: {
+    [ANNOTATION_PANEL.name]: {
+      fieldLayout: annotationFieldLayout([HIGH_IMPACT_GROUPS_SPLICE, MODERATE_IMPACT_GROUPS, CODING_IMPACT_GROUPS]),
     },
-    [DATASET_TYPE_SV_CALLS]: {
-      ...FREQUENCY_PANEL,
-      fields: SV_FREQUENCIES,
-      headerProps: {
-        ...FREQUENCY_PANEL.headerProps,
-        inputSize: 3,
-        inputProps: { component: SVFrequecyHeaderFilter },
-      },
+    [ANNOTATION_SECONDARY_NAME]: {
+      fieldLayout: annotationFieldLayout([HIGH_IMPACT_GROUPS, MODERATE_IMPACT_GROUPS, CODING_IMPACT_GROUPS]),
     },
-  },
-  LOCATION_PANEL_WITH_GENE_LIST,
-  {
-    ...QUALITY_PANEL,
-    [ALL_DATASET_TYPE]: {
-      ...QUALITY_PANEL,
-      fields: [...QUALITY_PANEL.fields, SV_QS_FILTER_FIELD, SV_GQ_FILTER_FIELD],
+    [IN_SILICO_PANEL.name]: {
+      fieldLayout: inSilicoFieldLayout(NO_SV_IN_SILICO_GROUPS),
+    },
+    [FREQUENCY_PANEL.name]: {
+      fields: SNP_FREQUENCIES,
     },
-    [DATASET_TYPE_SV_CALLS]: {
-      ...QUALITY_PANEL,
-      fields: [SV_QS_FILTER_FIELD, SV_GQ_FILTER_FIELD],
+    [QUALITY_PANEL.name]: {
+      fields: SNP_QUALITY_FILTER_FIELDS,
     },
   },
-]
+}
 
-const PANEL_MAP = [ALL_DATASET_TYPE, DATASET_TYPE_VARIANT_CALLS, DATASET_TYPE_SV_CALLS, ''].reduce((typeAcc, type) => {
-  const typePanels = PANELS.map(panel => (panel[type] === undefined ? panel : panel[type])).filter(panel => panel)
+const PANEL_MAP = [ALL_DATASET_TYPE, DATASET_TYPE_VARIANT_CALLS].reduce((typeAcc, type) => {
+  const typePanelProps = DATASET_TYPE_PANEL_PROPS[type] || {}
+  const typePanels = PANELS.map(panel => ({ ...panel, ...(typePanelProps[panel.name] || {}) }))
   return {
     ...typeAcc,
-    [type]: [true, false].reduce((analystAcc, hasHgmdBool) => {
-      const analystPanels = typePanels.map(
-        ({ hasHgmdPermission, ...panel }) => (hasHgmdPermission === undefined ? panel : hasHgmdPermission[hasHgmdBool]),
-      )
+    [type]: [true, false].reduce((hgmdAcc, hasHgmdBool) => {
+      const hgmdPanels = typePanels.map(panel => (
+        (!hasHgmdBool && panel.name === HGMD_PATHOGENICITY_PANEL) ? { ...panel, ...NO_HGMD_PANEL_PROPS } : panel
```

Comment:
changes how we set the hgmd/non-hgmd panel props so the main `PANELS` list can be a simple list of panels instead of having all these confusing dictionaries in it. Since HGMD is so specific to pathogenicity theres no need to support this as a generalizable configuration for all panels